src/main/java/lotto/week3/dto/WinningNumberRequestDto.java

```diff
@@ -0,0 +1,45 @@
+package lotto.week3.dto;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import lotto.week3.global.error.message.ErrorMessage;
+
+public class WinningNumberRequestDto {
+
+    private final List<Integer> winningNumbers;
+    private final int bonusNumber;
+
+    public WinningNumberRequestDto(List<Integer> winningNumbers, int bonusNumber) {
+        validateWinningNumbers(winningNumbers);
+        validateBonusNumber(winningNumbers, bonusNumber);
+        this.winningNumbers = winningNumbers;
+        this.bonusNumber = bonusNumber;
+    }
+
+
+    private void validateWinningNumbers(List<Integer> winningNumbers) {
+        if (winningNumbers.size() != 6) {
+            throw new IllegalArgumentException(ErrorMessage.WINNING_NUMBER_MUST_BE_SIX.getMessage());
+        }
+        Set<Integer> uniqueNumber = new HashSet<>(winningNumbers);
+        if (uniqueNumber.size() != 6 || uniqueNumber.stream().anyMatch(num -> num < 1 || num > 45)) {
```

Comment:
Lotto 클래스에서 MIN_NUMBER와 MAX_NUMBER를 상수로 선언하셨었는데,
다른 클래스에서 public으로 상수를 선언하셨었다면 이 부분에서 
MIN_NUMBER와 MAX_NUMBER로 활용하실 수 있을 것 같네요!
```suggestion
        if (uniqueNumber.size() != 6 || uniqueNumber.stream().anyMatch(num -> num < MIN_NUM || num > MAX_NUM)) {
```
이런식으로요!
이렇게 한다면 가독성이 올라갈 것 같습니다!

Reply:
으 실수 가 있었네요..